feature/system/tests/system_base_test/banner_test.go

```diff
@@ -97,31 +102,30 @@ func TestMotdBanner(t *testing.T) {
 // config_path:/system/config/login-banner
 // telemetry_path:/system/state/login-banner
 func TestLoginBanner(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
 	testCases := []struct {
 		description string
 		banner      string
 	}{
-		{"Empty String", ""},
-		{"Single Character", "x"},
-		{"Short String", "Warning Text"},
-		{"Long String", "WARNING : Unauthorized access to this system is forbidden and will be prosecuted by law. By accessing this system, you agree that your actions may be monitored if unauthorized usage is suspected."},
+		{"Empty String", deviations.BannerDelimiter(dut) + "" + deviations.BannerDelimiter(dut)},
+		{"Single Character", deviations.BannerDelimiter(dut) + "x" + deviations.BannerDelimiter(dut)},
+		{"Short String", deviations.BannerDelimiter(dut) + "Warning Text" + deviations.BannerDelimiter(dut)},
+		{"Long String", deviations.BannerDelimiter(dut) + "WARNING : Unauthorized access to this system is forbidden and will be prosecuted by law. By accessing this system, you agree that your actions may be monitored if unauthorized usage is suspected." + deviations.BannerDelimiter(dut)},
 	}
-
-	dut := ondatra.DUT(t, "dut")
-
 	for _, testCase := range testCases {
 		t.Run(testCase.description, func(t *testing.T) {
 			config := gnmi.OC().System().LoginBanner()
 			state := gnmi.OC().System().LoginBanner()
-
 			gnmi.Replace(t, dut, config.Config(), testCase.banner)
 
 			t.Run("Get Login Banner Config", func(t *testing.T) {
-				if testCase.banner == "" {
-					if gnmi.LookupConfig(t, dut, config.Config()).IsPresent() {
-						t.Errorf("Config Login Banner not empty")
-					} else {
-						t.Logf("No response for the path expected is expected as the config is empty")
+				if testCase.description == "Empty String" {
+					if dut.Vendor() != ondatra.CISCO {
```

Comment:
Same comments as above, do not use vendor matches or matches on test case descriptions. This should all be gated by deviations.